NitroxLauncher/Pages/ServerPage.xaml.cs

```diff
@@ -1,37 +1,850 @@
+            {
+                TBSaveInterval.Text = originalSaveInterval;
+                LauncherNotifier.Error($"An empty Save Interval value is not valid.");
+                return;
+            }
+
+            int SaveIntervalNum;
+            try
+            {
+                SaveIntervalNum = Convert.ToInt32(TBSaveInterval.Text)*1000;
+            }
+            catch
+            {
+                TBSaveInterval.Text = originalSaveInterval;
+                LauncherNotifier.Error($"Save Interval input should only contain numbers.");
+                return;
+            }
+
+            // Limit save interval value to numbers greater than 1
+            if (SaveIntervalNum < 1)
+            {
+                TBSaveInterval.Text = originalSaveInterval;
+                LauncherNotifier.Error($"The Save Interval value must be greater than 1.");
+                return;
+            }
+
+            Config.SaveInterval = SaveIntervalNum;
+        }
+
+        private void TBJoinPassword_Input(object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
+        {
+            TBJoinPassword.Text = TBJoinPassword.Text.TrimStart();
+            TBJoinPassword.Text = TBJoinPassword.Text.TrimEnd();
+
+            if (string.IsNullOrEmpty(TBJoinPassword.Text))
+            {
+                CBEnableJoinPassword.IsChecked = false;
+                TBJoinPassword.IsEnabled = false;
+                JoinPasswordTitle.Opacity = .6;
+            }
+
+        }
+
+        private void TBWorldServerPort_Input(object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
+        {
+            string originalServerPort = Convert.ToString(Config.ServerPort);
+
+            TBWorldServerPort.Text = TBWorldServerPort.Text.TrimStart();
+            TBWorldServerPort.Text = TBWorldServerPort.Text.TrimEnd();
+
+            if (string.IsNullOrEmpty(TBWorldServerPort.Text))
+            {
+                TBWorldServerPort.Text = originalServerPort;
+                LauncherNotifier.Error($"An empty Server Port value is not valid.");
+                return;
+            }
+
+            int ServerPortNum;
+            try
+            {
+                ServerPortNum = Convert.ToInt32(TBWorldServerPort.Text);
+            }
+            catch
+            {
+                TBWorldServerPort.Text = originalServerPort;
+                LauncherNotifier.Error($"Server Port input should only contain numbers.");
+                return;
+            }
+            
+            // Limit the input to numbers in between ports 1024 and 65535
+            if (ServerPortNum < 1024 || ServerPortNum > 65535)
+            {
+                TBWorldServerPort.Text = originalServerPort;
+                LauncherNotifier.Error($"Only port numbers between 1024 and 65535 are allowed.");
+                return;
+            }
+
+            Config.ServerPort = ServerPortNum;
+        }
+
+        // TODO
```

Comment:
```suggestion
        // TODO: Redirect user to the "Mods/Plugins" tab of the launcher (for future reference if mod support is added) so that they can enable/disable mods
```